src/parsers/vmap/vmap.js

```diff
@@ -58,7 +57,7 @@ export default class VMAPManager {
           category = 'midroll'
         }
 
-        const adBreak = this.createAdBreak({
+        const adBreak = this.formatTimeOffset({
```

Comment:
I think it's better to use the `formatTimeOffset` return value just timeOffset. Because I understand that it isn't his responsibility to create the parsed adBreak object.
Like as:
`const adBreak = {
    category,
    adTag: item['vmap:AdSource']['vmap:AdTagURI'],
    timeOffset: formatTimeOffset(timeOffset)
  }`
Or create a variable to receive the new formatted value and use it in the adBreak object.
What do you think?